CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ### Added
 
+- Add support for .NET 7.
```

Comment:
The plan was only to add tests for .NET 7, but additional dependencies/store requires separate folder structure for each supported .NET version.